src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -521,6 +521,7 @@ private function readInput($inputStream, Question $question)
         while (false !== ($char = fgetc($inputStream))) {
             $ret .= $char;
         }
+        $this->inputStream = fopen('php://stdin', 'r');
```

Comment:
what if the input stream being used is *not* `php://stdin` ? That's a supported use case (see `isInteractiveInput` supporting that case explicitly for instance).

Also, this method looks like the wrong place to me, as it is not designed to use `$this->inputStream` directly.

Reply:
That's a very valid point.
I have no idea how to deal with it ^^

Reply:
I think I have a solution. I'm testing it on Windows to make sure it works there, as well.